src/megameklab/com/ui/MegaMekLabMainUI.java

```diff
@@ -112,10 +158,14 @@ public void exit() {
             CConfig.setParam("WINDOWSTATE", Integer.toString(getExtendedState()));
             // Only save position and size if not maximized or minimized.
             if (getExtendedState() == Frame.NORMAL) {
+                /*
```

Comment:
Why change this behavior? I consider being able to set the window size and location and have it come back in the same state the next time I run it to be desirable.

Reply:
The loading dialog now sets the window to be of large enough size to fit the whole tabbedpane without scrollbars (unless screen is too small) and centers it on the screen. I suppose we could leave the X and Y parameters here for placement, but my concern with saving the size components, is that the windows are different sizes for different unit types. So you might end up opening a window that is too small for the chosen unit.

Reply:
I work on a laptop with a max display resolution of 1366x768, so I don't have the luxury of being averse to scrollbars. I do sometimes size and position a window to take up part of the screen so I can see another window at the same time (in this case perhaps MML and a TRO). As long as I'm working with the same unit type it's not a problem, but if I switch unit types I think it's reasonable to expect MML not to override my choices and force me to go through the process of resizing the window repeatedly. I consider that a basic aspect of usability.

Reply:
So after doing some work with the master branch I discovered that it actually doesn't do this now, and this whole line of argument is irrelevant. Never mind.